test_component/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+{'name': 'Components Tests',
+ 'summary': 'Automated tests for Components, do not install.',
```

Comment:
Hi @guewen ,

Could you add a description tag so that Travis pass all the build ?